pkg/devcontainer/config.go

```diff
@@ -39,7 +39,7 @@ func (r *runner) getRawConfig(options provider2.CLIOptions) (*config.DevContaine
 			return nil, err
 		}
 
-		return config.ParseDevContainerJSON(
+		return config.Parse(
```

Comment:
same as above, not sure how we differentiate between `config.Config` and this devcontainer `config.Config`